Tutorial_5_BonusNotebook.ipynb

```diff
@@ -0,0 +1,756 @@
+    "  <summary>Click for help</summary>\n",
+    "   \n",
+    "\n",
+    "`for x in `cat accs.txt`; do fasterq-dump -f -O data/raw_fastq -e $CORES -m 4G data/raw_fastq/$x/$x.sra; done`  \n",
+    "\n",
+    "   \n",
+    "  \n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Convert to fastq.gz"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "   \n",
+    "\n",
+    "`pigz data/raw_fastq/*.fastq`  \n",
+    "\n",
+    "   \n",
+    "  \n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### STEP 4: Copy reference transcriptome files that will be used by Salmon using E-Direct\n",
+    "\n",
+    "As before, we are going to use Salmon to quantify the transcriptome. We will need a Refseq assembly here. Since these reads are on the _Plasmodium falciparum_ organism, we need to update our Refseq files. Try searching through [NCBI genome database](https://www.ncbi.nlm.nih.gov/datasets/genome) for the appropriate _Plasmodium falciparum_ assembly.\n",
+    "\n",
+    "Once you find an assembly, download transcriptome reference files from that assembly using FTP links.\n",
+    "\n",
+    "Try repurposing the esearch commands from Tutorial1B to download the files for _P. falciparum_. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "<YOUR COMMAND HERE>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "  <summary>Click for help</summary>\n",
+    "   \n",
+    "\n",
+    "`#parse for the ftp link and download the genome reference fasta file`  \n",
+    "`esearch -db assembly -query GCF_000002765.6 | efetch -format docsum \\`  \n",
+    "`| xtract -pattern DocumentSummary -element FtpPath_RefSeq \\`  \n",
+    "`| awk -F\"/\" '{print \"curl -o data/reference/\"$NF\"_genomic.fna.gz \" $0\"/\"$NF\"_genomic.fna.gz\"}' \\`  \n",
+    "`| bash`  \n",
+    "\n",
+    "`#parse for the ftp link and download the gtf reference fasta file`  \n",
+    "`esearch -db assembly -query GCF_000002765.6 | efetch -format docsum \\`  \n",
+    "`| xtract -pattern DocumentSummary -element FtpPath_RefSeq \\`  \n",
+    "`| awk -F\"/\" '{print \"curl -o data/reference/\"$NF\"_genomic.gff.gz \" $0\"/\"$NF\"_genomic.gff.gz\"}' \\`  \n",
+    "`| bash`  \n",
+    "\n",
+    "`# parse for the ftp link and download the feature-table reference file`  \n",
+    "`# (for later use for merging readcounts with gene names in R code).`\n",
+    "`esearch -db assembly -query GCF_000002765.6 | efetch -format docsum \\`  \n",
```

Comment:
put a space between the commend and esearch on this line for some reason when I copy the code it becomes part of the comment.

Reply:
Fixed. There are newlines now and the comments are regular markdown.